build_scripts/windows/Product.wxs

```diff
@@ -54,14 +93,14 @@
     <!-- Show message to restart any terminals only if the PATH is changed -->
     <CustomAction Id="Set_WIXUI_EXITDIALOGOPTIONALTEXT" Property="WIXUI_EXITDIALOGOPTIONALTEXT" Value="Please close and reopen any active terminal window to use Azure CLI." />
     <InstallUISequence>
-      <Custom Action="Set_WIXUI_EXITDIALOGOPTIONALTEXT" After="CostFinalize">NOT Installed AND NOT WIX_UPGRADE_DETECTED</Custom>
+      <Custom Action="Set_WIXUI_EXITDIALOGOPTIONALTEXT" After="CostFinalize">NOT Installed AND NOT WIX_UPGRADE_DETECTED AND NOT WIX_X86_UPGRADE_DETECTED</Custom>
```

Comment:
I think `WIX_X86_UPGRADE_DETECTED` does change the `PATH` from `C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\wbin` to `C:\Program Files\Microsoft SDKs\Azure\CLI2\wbin` , so I guess we do need to show this message?

```suggestion
      <Custom Action="Set_WIXUI_EXITDIALOGOPTIONALTEXT" After="CostFinalize">NOT Installed AND NOT WIX_UPGRADE_DETECTED</Custom>
```